site-src/blog/2023/0823-welcome-gamma.md

```diff
@@ -0,0 +1,160 @@
+---
+description: >
+  We are excited to announce the v0.8.0 release of Gateway API, where the GAMMA
+  initiative has now reached Experimental status, Gateway and HTTPRoute gain
+  some new capabilities, conformance profiles are supported, and more!
+---
+
+# Gateway API: Welcome, GAMMA!
```

Comment:
I expect a lot of potential readers of this post will not know what GAMMA is, maybe we should just use "Service Mesh" in the title and then explain what GAMMA is in the blog post.

```suggestion
# Gateway API: Welcome, Service Mesh!
```